apps/node-pl/pattern-lab/_patterns/01-atoms-demo/svg/svgs.twig

```diff
@@ -2,9 +2,106 @@
   demo: svg_demo_block
 } %}
 
-  {% block content %}
-    TO BE CREATED
+	{% block content %}
 
-  {% endblock %}
+		<h1 class="mt-2">Embed</h1>
+    <p class="mb-2">Includes the entire SVG markup within the DOM. This is the preferred method.</p>
+		<div class="flex">
+      <!--
+        Embed: The minimum required information. Note the FULL namespaced path
+        to to the icon svg. Width here is set by a WRAPPING <div>.
+      -->
+			<div class="w-10">
+				{% include '@atoms/svg/_svg.twig' with {
+          svg: {
+            path: '@atoms/svg/icons/person.svg',
+          }
+        } %}
+			</div>
+      <!--
+        Embed: Width is still set via a wrapping div, and so is color. Note:
+        set color via a TEXT color class, due to embedded SVGs defaulting to
+        fill: currentColor;
+      -->
+			<div class="w-6 text-blue-500">
+				{% include '@atoms/svg/_svg.twig' with {
+          svg: {
+            path: '@atoms/svg/icons/close.svg',
+          }
+        } %}
+			</div>
+      <!--
+        Embed: Leveraging additional classes being passed directly to the SVG
+        component, no need for an extra wrapping <div>
+      -->
+      {% include '@atoms/svg/_svg.twig' with {
+        svg: {
+          path: '@atoms/svg/icons/search.svg',
+        },
+        svg_classes: ['text-red-500', 'w-4'],
```

Comment:
Colors are a really likely point of Tailwind config we'll remove. I've tried to only use `gray` in docs with the thought that once we remove colors this is a point of breakage. We might consider skipping colors in demo.

Reply:
"Before removing colors in tailwind, ensure you command+f and search for any of the removed colors, ie `red-500`"